speech/google/cloud/speech/streaming_response.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Representation of a GAPIC Speech API response."""
+
+from google.cloud.speech.streaming_result import StreamingSpeechResult
+from google.cloud.gapic.speech.v1beta1.enums import StreamingRecognizeResponse
+
+
+class StreamingSpeechResponse(object):
+    """Representation of a Speech API protobuf streaming response.
+
+    :type error: :class:`google.grpc.Status`
+    :param error: Instance of ``Status``
+
+    :type endpointer_type: :class:`~EndpointerType`
+    :param endpointer_type: Enum of endpointer event.
```

Comment:
Instances of this class don't actually have any value. And reverse map expects integers -> string? As in https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2523#discussion_r83702766

> If we only wanted a string anyway, let's just provide the strings as nice constants that are easy to access?